docs/src/main/paradox/kinesis.md

```diff
@@ -60,10 +60,10 @@ Java
 The `KinesisSource` creates one `GraphStage` per shard. Reading from a shard requires an instance of `ShardSettings`. 
 
 Scala
-: @@snip (../../../../kinesis/src/test/scala/akka/stream/alpakka/kinesis/scaladsl/Examples.scala) { #settings }
+: @@snip (../../../../kinesis/src/test/scala/akka/stream/alpakka/kinesis/scaladsl/Examples.scala) { #source-settings }
```

Comment:
Would be nice to add the root of the project as a variable like we do for akka, but that's for another PR